Doc/whatsnew/3.14.rst

```diff
@@ -408,6 +408,8 @@ http
   module allow the browser to apply its default dark mode.
   (Contributed by Yorik Hansen in :gh:`123430`.)
 
+* Added support for HTTP Range header to :class:`~http.server.HTTPServer`.
```

Comment:
If possible, let's also add something like "as specified in :rfc:\`ABC, §X.Y.Z.T <ABC#X.Y.Z.T>\`" (same for the NEWS entry)